checkov/sca_package/scanner.py

```diff
@@ -127,18 +127,30 @@ def run_scan_busy_wait(self, input_path: Path, scan_id: str) -> dict[str, Any]:
                 logging.error(response_json["outputData"])
                 return {}
 
+            time.sleep(SLEEP_DURATION)
```

Comment:
why are we adding a sleep?

Reply:
@schosterbarak actually this sleep wasn't added , it just moved a few lines above. We sleep between calls in the original version as well.